spec/models/vote_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Vote, type: :model do
+  #pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
New model = new model tests! I'd expect to see tests for any validations and associations in your model here. Shouldamatchers should provide helpers that make this easy. Here's a good example of how model specs often end up looking: https://github.com/vigetlabs/teach_chemistry/blob/master/spec/models/group_spec.rb